docs/src/token-2022/extensions.md

```diff
@@ -2,4 +2,997 @@
+
+    const connection = new Connection(clusterApiUrl('devnet'), 'confirmed');
+
+    const airdropSignature = await connection.requestAirdrop(payer.publicKey, 2 * LAMPORTS_PER_SOL);
+    await connection.confirmTransaction(airdropSignature);
+
+    const mintLamports = await connection.getMinimumBalanceForRentExemption(mintLen);
+    const mintTransaction = new Transaction().add(
+        SystemProgram.createAccount({
+            fromPubkey: payer.publicKey,
+            newAccountPubkey: mint,
+            space: mintLen,
+            lamports: mintLamports,
+            programId: TOKEN_2022_PROGRAM_ID,
+        }),
+        createInitializeTransferFeeConfigInstruction(
+            mint,
+            transferFeeConfigAuthority.publicKey,
+            withdrawWithheldAuthority.publicKey,
+            feeBasisPoints,
+            maxFee,
+            TOKEN_2022_PROGRAM_ID
+        ),
+        createInitializeMintInstruction(mint, decimals, mintAuthority.publicKey, null, TOKEN_2022_PROGRAM_ID)
+    );
+    await sendAndConfirmTransaction(connection, mintTransaction, [payer, mintKeypair], undefined);
+})();
+```
+
+  </TabItem>
+</Tabs>
+
+#### Example: Transferring tokens with the fee checked
+
+As part of the extension, there is a new `transfer_checked_with_fee` instruction,
+which accepts the expected fee. The transfer only succeeds if the fee is correctly
+calculated, in order to avoid any surprises during the transfer.
+
+<Tabs className="unique-tabs" groupId="language-selection">
+  <TabItem value="cli" label="CLI" default>
+
+CLI support coming soon!
+
+  </TabItem>
+  <TabItem value="jsx" label="JS">
+
+```jsx
+    const mintAmount = BigInt(1_000_000_000);
+    const owner = Keypair.generate();
+    const sourceAccount = await createAccount(
+        connection,
+        payer,
+        mint,
+        owner.publicKey,
+        undefined,
+        undefined,
+        TOKEN_2022_PROGRAM_ID
+    );
+    await mintTo(
+        connection,
+        payer,
+        mint,
+        sourceAccount,
+        mintAuthority,
+        mintAmount,
+        [],
+        undefined,
+        TOKEN_2022_PROGRAM_ID
+    );
+
+    const accountKeypair = Keypair.generate();
+    const destinationAccount = await createAccount(
+        connection,
+        payer,
+        mint,
+        owner.publicKey,
+        accountKeypair,
+        undefined,
+        TOKEN_2022_PROGRAM_ID
+    );
+
+    const transferAmount = BigInt(1_000_000);
+    const fee = (transferAmount * BigInt(feeBasisPoints)) / BigInt(10_000);
+    await transferCheckedWithFee(
+        connection,
+        payer,
+        sourceAccount,
+        mint,
+        destinationAccount,
+        owner,
+        transferAmount,
+        decimals,
+        fee,
+        [],
+        undefined,
+        TOKEN_2022_PROGRAM_ID
+    );
+```
+
+  </TabItem>
+</Tabs>
+
+#### Example: Withdraw withheld tokens from accounts
+
+As users transfer their tokens, transfer fees accumulate in the various recipient
```

Comment:
An additional example that would be useful: How do I get a list of accounts holding fees?

Reply:
There's no really easy way unfortunately... you'd have to iterate through the accounts and check them one by one.  Either way, I'll add it.